### ISSUE TITLE
Confirmation of COVID-19 not possible

### ISSUE THREAD

**Issue by nikola** [jgehrcke/covid-19-germany-gae#53]
As of this writing, March 27th, 2020, there is actually no scientific test - as in: a provable and repeatable test - to confirm a case of COVID-19. Here's what the Wikipedia community says:

https://en.wikipedia.org/wiki/Coronavirus_disease_2019#Diagnosis

First the ambiguous part which is probably misleading a large part of the public into thinking that COVID-19 can be tested for:

> The WHO has published several testing protocols for the disease. The standard method of testing is real-time reverse transcription polymerase chain reaction (rRT-PCR). The test can be done on respiratory samples obtained by various methods, including a nasopharyngeal swab or sputum sample.[62] Results are generally available within a few hours to two days. Blood tests can be used, but these require two blood samples taken two weeks apart and the results have little immediate value.

This is apparently making many readers believe that the paragraph is describing the diagnosis of COVID-19. It's not, because the clarification is right in the next sentence, emphasis mine:

> Chinese scientists were able to isolate a strain of the coronavirus and publish the genetic sequence so that laboratories across the world could independently develop **polymerase chain reaction (PCR) tests to detect infection by the virus.** As of 19 March 2020, there were no antibody tests though efforts to develop them are ongoing.

There is no test for COVID-19 (the disease) but a test for SARS-CoV-2 (the virus).

So what are symptoms of COVID-19?

> Diagnostic guidelines released by Zhongnan Hospital of Wuhan University suggested methods for detecting infections based upon clinical features and epidemiological risk. These involved identifying people who had at least two of the following symptoms in addition to a history of travel to Wuhan or contact with other infected people: fever, imaging features of pneumonia, normal or reduced white blood cell count, or reduced lymphocyte count.

In other words, a wide range of symptoms caused by anything from bacteria to viruses.

Moreover:

> One study in China found that CT scans showed ground-glass opacities in 56%, but 18% had no radiological findings. Bilateral and peripheral ground glass opacities are the most typical CT findings, though they are non-specific.

So these are non-specific, but even aside of that: no country has such a large capacity of CT equipment to test each suspected case of COVID-19.

Add to this that the vast majority of published and claimed COVID-19 cases had one or more pre-existing conditions which either weaken the immune-system or attack the lungs, i.e. conditions which would also cause the above symptoms: fever, imaging features of pneumonia, normal or reduced white blood cell count, or reduced lymphocyte count.

I repeat: right now there is no way to prove in any scientific sense that a person infected with SARS-CoV-2 and who also developed e.g. pneumonia is actually infected by COVID-19, or if they were "only" infected by SARS-CoV-2, fought off the infection by SARS-CoV-2 but developed the above symptoms due to an unrelated infection with other bacteria or viruses. Which would actually explain why the vast majority of people infected by SARS-CoV-2 survive without developing any of the above symptoms, and why the vast majority of confirmed or suspected COVID-19 cases had pre-existing conditions, namely conditions causing the exact same symptoms.

Here's how the Robert-Koch-Institute confuses the situation even more:

https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Situationsberichte/2020-03-26-en.pdf?__blob=publicationFile

> Clinical aspects
> Clinical information is available for 26,250 of the notified cases, of which 870 cases were reported as not having any symptoms considered significant for COVID-19. The most common manifestations are cough (14,202; 54%), fever (10,784; 41%), rhinorrhoea (6,158; 23%) and pneumonia (429; 2%). Hospitalisation was reported in 2,664 (10%) of the 26,563 COVID-19 cases with data available. An estimated 5,900 persons have recovered from their COVID-19 infection. Cases were considered to have recovered if they had a known onset of disease on or before 12/03/2020, were not reported to have pneumonia or dyspnea, did not require hospitalisation or had already been discharged and did not die. Cases were included in the algorithm only if information on date of illness onset, symptoms, hospitalisation status and vital status were available. 

So the RKI is counting people infected by SARS-CoV-2 who had no symptoms of pneumonia or dyspnea and who were not hospitalised, as cases of cured COVID-19, instead of counting them as cases of SARS-CoV-2 infection without developing COVID-19. I guess that's called proof-through-absence-of-evidence? But don't take my word for it, here's what the RKI itself admits in the German-only case definition file, emphasis mine:

https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Falldefinition.pdf?__blob=publicationFile

> Epidemiologische Bestätigung
> Epidemiologische Bestätigung, definiert als **mindestens einer** der beiden folgenden Nachweise unter Berücksichtigung der Inkubationszeit:
> * epidemiologischer Zusammenhang mit einer labordiagnostisch nachgewiesenen Infektion beim Menschen durch - Mensch-zu-Mensch-Übertragung
> * Auftreten von zwei oder mehr Lungenentzündungen (Pneumonien) (spezifisches klinisches Bild) in einer medizinischen Einrichtung, einem Pflege- oder Altenheim, bei denen ein epidemischer Zusammenhang wahrscheinlich ist oder vermutet wird, **auch ohne Vorliegen eines Erregernachweises**.

Please read the last part carefully:

> bei denen ein epidemischer Zusammenhang wahrscheinlich ist oder vermutet wird, auch ohne Vorliegen eines Erregernachweises.

in English: a COVID-19 infection is to be treated as confirmed if there were two or more **cases of pneumonia** in a medical facility, nursing home or retirement home, where an epidemiological connection is probable or **suspected**, **even if there is no positive test result for the virus**.

And here is what the RKI says which cases should be reported as confirmed cases of COVID-19, emphasis mine:

> Über die zuständige Landesbehörde an das RKI zu übermittelnder Fall
> B. Klinisch-epidemiologisch bestätigte Erkrankung
> Spezifisches klinisches Bild von COVID-19, **ohne labordiagnostischen Nachweis**, aber mit epidemiologischer Bestätigung (Auftreten von zwei oder mehr Lungenentzündungen (Pneumonien) in einer medizinischen Einrichtung, einem Pflegeoder Altenheim).
> Spezifisches oder unspezifisches klinisches Bild von COVID-19, ohne labordiagnostischen Nachweis, aber mit epidemiologischer Bestätigung (Kontakt zu einem bestätigten Fall).

In English: all cases of pneumonia where two or more cases of pneumonia occured in a medical facility, nursing home or retirement home where an epidemiological connection is established should be transmitted by state authorities to the RKI as a case of COVID-19 **even in the absence of a positive test for SARS-CoV-2** in the patient who developed the pneumonia.

So given the fact that:

+ there is no region left in Germany where no person has tested positive for SARS-CoV-2, and
+ therefore the condition that an epidemiological connection must exist is always true for every medical facility, nursing home or retirement home in the country,

the RKI is basically saying that starting in March 2020 all cases of pneumonia in Germany should be reported as COVID-19. Imagine that.

Therefore I think this project, useful as it may be, should correctly label the findings: if they are really just giving positive test numbers for SARS-CoV-2 then it should read that this is what the findings show, nothing more. The numbers should also point out the above: cases of unrelated pneumonia with no testing done for SARS-CoV-2 where the sick person was somehow connected to a confirmed case of SARS-CoV-2 are (mis-)represented as a confirmed case of COVID-19.

**Comment by nikola** [comment]
A careful reader might have noticed that this doesn't touch the subject of alleged cases of COVID-19 where it's pneumonia and a test was done for SARS-CoV-2 that came back negative. Here's what the RKI says about those:

> Ein negatives PCR-Ergebnis schließt die Möglichkeit einer Infektion mit SARSCoV-2 nicht vollständig aus. Falsch-negative Ergebnisse können z.B. aufgrund schlechter Probenqualität, unsachgemäßem Transport oder ungünstigem Zeitpunkt (bezogen auf den Krankheitsverlauf) der Probenentnahme nicht ausgeschlossen werden.

In English: false negatives can occur in PCR tests for SARS-CoV-2.

In the context of the above it means that medical facilities, nursing homes and retirement homes are invited to report a case of pneumonia as COVID-19 even if the test for SARS-CoV-2 was negative as long as there is an epidemiological connection for the facility, which is true for all facilities in Germany as of this writing because they are all in regions where at least one person tested positive for SARS-CoV-2.

**Comment by jgehrcke** [comment]
> Therefore I think this project, useful as it may be, should correctly label the findings: if they are really just giving positive test numbers for SARS-CoV-2 then it should read that this is what the findings show, nothing more. 

Thanks. This is important, and should indeed be known and obvious to all people consuming numbers like those in this dataset.

Yesterday I did actually put the following sentence into the README, in the section titled *What you should know before reading these numbers* (you have probably not seen it yet?):

> We believe that each "confirmed case" actually corresponds to a polymerase chain reaction (PCR) test for the SARS-CoV2 virus with a positive outcome. This is quite probably true, but we cannot verify this end-to-end, we have to trust Landkreise, doctors, and labs.

Beyond that, the overall disclaimer is in the README that one better should _think_ before interpreting these numbers deeply. Hope that satisfies you :). I for my part appreciate that you take this with a lot of care, and I also believe that a disclaimer like this is already much better than most other places presenting data like this.

Other than that, what is your high-level conclusion? If you had to write an abstract of 200 words -- which words would you put into it? Genuinely curious.

Regardless of testing deficiencies and typical correlation-causality issues, **I do hope that we don't neglect the exceptional real-world events happening around us right now**, where various countries and cities have collapsing health systems, where Italien children are hospitalized in Hannover, where French people are hospitalized in Germany, where the New York City hospitals are beyond capacity already today, where Spain, Italy, France have accumulated dead bodies at a rate that their crematories cannot keep up with, and where we have [reports like this](https://www.zeit.de/wissen/gesundheit/2020-03/clemens-wendtner-coronavirus-covid-19-intensivstationen-beatmung-krankenhaus) from clinics in Germany, [endless list of things happening around us right now, and even more brutal things likely to happen in the near future].

**Comment by nikola** [comment]
Fear is not evidence that an event is happening. Substantiated case numbers are evidence that an event is happening.

No one is disputing that SARS-CoV-2 is real, or that COVID-19 is real, or that COVID-19 is a disease with a higher lethality than your regular influenza. The issue here is the scale, i.e. how many cases of real COVID-19 are there actually, and clearly many stakeholders around the world are keen to inflate these case numbers to stoke fear and subsequently enforce their personal agenda which may be disparate but ultimately converges into measures we're now seeing implemented worldwide. With the most simple and obvious agenda being that across the planet the vast majority of power is concentrated into the hands of old people, people who can buy everything but not immunity to death, and who are now afraid that even a mortality rate of 0.1% might claim their own life because SARS-CoV-2 is clearly more contagious than influenza or other viruses. I'm not judging, just stating a simple fact.

As a German you've probably heard plenty of statements from Merkel, 66, clearly not leading a healthy lifestyle. Did she ever say, "Out of my personal fear we now must act"? No, instead she's saying, "The hard numbers supplied by the Robert-Koch-Institute force our hand."

I find it ironic that after years of campaigning the public to question every statistic that's not properly sourced and verified the same people are now selling misconstrued numbers as reality.

But it's even more ironic that for some publications in Germany, most notably SPIEGEL, even that's not enough, and just days after the RKI has been elevated by the government as a new national body for coordinating the fight against this epidemic, SPIEGEL is now saying that the misconstrued numbers by RKI are unreliable and that we should be believing the - much higher! - JHU numbers instead, numbers that come out of nowhere and which have no attribution at all. If that doesn't tell you what the agenda is then I don't know what will.

Regarding your link, I take it that you have no medical background. Let me tell you as someone who has many doctors and nurses across relatives and friends that those articles are squarely aimed at the general public who also have no medical background or access to immediate information. It's been reality since forever that you'll always find an institution struggling with a particular wave of conditions. The vast majority of German hospitals have plenty of excess capacity, including spare ventilators, leaving aside the question why there're not sending more of them to Italy or France, but I guess that's a question that only non-selfish Ms Merkel can answer. Also, it appears that you are thinking intensive care or mechanical ventilation are extraordinary measures unheard of in normal times. They are not. Every caretaker (Altenpfleger) I know would not be challenged at all to apply ventilation to a patient in need. It's a standard procedure unrelated to COVID-19, just as pneumonia is a standard condition in winter months in the northern hemisphere, unfortunately. While I was typing this, an average of 3 people died from pneumonia in Germany alone, 50.000 cases per year, and even more when a severe influenza epidemic strikes.

Based on RKI's approach of counting every case of pneumonia as COVID-19, there's your number of expected fatalities, and it's approximately this number that RKI will publish as the final death toll of SARS-CoV-2/COVID-19 by the end of the year. See, I can make bold predictions that might never come true, just as all the virologists that travel from format to format and column to column.

**Comment by nikola** [comment]
> Other than that, what is your high-level conclusion? If you had to write an abstract of 200 words

If I had to write an abstract I'd write a prediction that ... oh, wait, something else noticed the same thing:

https://www.welt.de/wirtschaft/article206843189/Corona-Krise-Das-RKI-laesst-die-Menschen-allein.html

... that soon more than a few people will notice that the official numbers published by RKI make no sense in a virological, epidemiological or mathematical sense, and that the RKI would first backtrack, then hide in the shadows, especially once it becomes clear that they never had an actual plan to see this through. That's why RKI has stopped inviting journalists to the press conferences, or even responding to questions fielded, as the article points out.

I believe the first step should've been to isolate and care for the high-risk population - elderly and/or serious existing conditions -, second step to test this group starting from highest to lowest risk, but at the same time issue thermometers to each household and ask them to input each day temperature readings for each member of the household (if needed with assistance from caretakers), to identify hotspots in the non-isolated population. Once identified, isolate and treat all infected.

That's not even my own idea. Kinsa Inc. is doing this on a much more sophisticated level with an app and apparently a custom device, but the solution I outlined above is low-tech, low-cost and works everywhere. You can even start doing this today, it's not too late, and costs a fraction of the money they're pumping into the economy due to the damage they caused with RKI's "plan" (in quotes).

Because that's the point I was trying to make earlier, and something I noticed early on: the clinical symptoms make no sense, it should be sufficient to check for fever, pneumonia and other symptoms never occur without fever, on the other hand you might have fever from COVID-19 but no pneumonia.

Checking for fever was also precisely the method China and South Korea used and are still using to identify potential outbreaks of the disease.

**Comment by nikola** [comment]
The result of my approach would look something like this:

https://healthweather.us/

What's also interesting about the current readings:

![image](https://user-images.githubusercontent.com/54608/77798773-d37dac00-7073-11ea-861a-0c2577301602.png)

you can clearly make out the hotspots on the east coast, but also the belt in the midwest from north to south. Given that lots of the usual transit has been reduced, this looks like a wave of human-to-human transmission travelling from east coast to west coast.

**Comment by nikola** [comment]
Ironically, these days when random people input bogus information into shared spreadsheets it's called crowdsourcing. It's not. What I've shown above is, because the actual information comes from the test subjects directly. No hearsay, no double-reporting. And given that every resident in Germany was issued a unique ID (Steueridentifikationsnummer), it would be very easy to write a web application for capturing this data, even anonymized: through a hash function that turns the Steueridentifikationsnummer into a unique tag, e.g. through AES3, for all cases where the temperature reading triggers no warning but where government still wants to collect and possibly share this data with researchers. And for cases where the reading indicates an outbreak of COVID-19 it would be easy for the government to resolve the ID into an actual person, which is already covered by infection and privacy laws.

There. Solved. Saves countless lives, and we don't even have to bankrupt thousands of companies.

**Comment by jgehrcke** [comment]
#### About this repo

A few general points about the scope of responsibility of this repository, and about the source and destination of the numbers presented by it:

1. One of the very localized problem statements behind this repository: make the COVID-19 case numbers _as determined by the German health care system_ available through common interfaces.
2. Certainly, discussing and improving the method for determining these numbers is an important, ongoing effort. That effort is not part of this repository.
3. Certainly, the interpretation of these numbers is an ongoing effort that is not part of this repository. As of today, personally, I very much agree that these numbers alone are not suitable for answering about any relevant question that seeks answers _right now_. The disclaimer in the README clarifies that (hopefully).
4. Also, do believe that these numbers have the potential to mislead the uninformed public. Personally, however, I also believe that curious, motivated, positively-thinking people should have access to these numbers. I also believe that (2) will get better over time, and that we will also gain much more clarity around (3) over time in which case a repository like this will become more valuable. Limited use right now, yes, but think of it as (an optimistic attempt of) laying ground work.

#### A personal note about the form you write in

I want zoom out and comment a bit about what you wrote and how you wrote it.

I do believe that we both understand that this discussion is rather off-topic from this repository (I really value point 1 in the list above), and we both know that it is a very deep topic. Therefore, I do hope that you are not disappointed when I invest only little energy and time on this thread. There are better places to discuss this.

The most productive part of my response I think can only be on a personal level; with some honest, genuine feedback.

To your high-level claims: I actually still don't really know what you claim, at least I don't know what the core message of yours is; what your major conclusions are. If you want to be heard and if you would like to have a productive discussion maybe you should really try summarizing your major claims first, and then later elaborate more in-depth. Such a format could be easier to consume.

> If I had to write an abstract I'd write a prediction

:-(. My question for an abstract was absolutely genuine, and well-intended.

If you want to be heard and if you would love to engage in a productive discussion I can also recommend a little bit of a more positive tone. Much of what you write here is a challenging to follow because of the amount of text, the (to me, at least) unstructured order of small-ish claims, and a bit of a negative tone, to be honest.

#### About the matter itself (high-level)

> Substantiated case numbers are evidence that an event is happening

I wholeheartedly agree with that statement. And I am optimistic, even certain, that decision-makers in Germany agree with this. That the RKI's mode of operation is _lead_ by this, it's their north star. You have not managed to explain to me _why_ you believe that they are _not_ operating in this mode.

Please consider that three weeks ago, even one week ago, "substantiated case numbers" as in "bulletproof statistical conclusions" were simply not available, **yet** decisions had to be made. The RKI and its consultants and the government have at all points in time clarified exactly that.

One of the people who undeniably has had a strong influence on political decisions and public perception is Christian Drosten. I have tried to listen to many of his public statements. Coming from academia myself: to my taste, he operates with a rather satisfying extent of scientific care, and pays a keen eye to detail. The extent to which he says "we don't know" satisfies my scientific skepticism. As far as I can tell, he's a true defender of the scientific method as we know it. As an _example_, just yesterday in his podcast series on NDR info ([here, from March 27](https://www.ndr.de/nachrichten/info/podcast4684.html)) he explains in-depth how little we understand about COVID-19 as a disease, how deficient the testing system is to date, and how German university clinics have barely started to trying to shed some light onto this.

In terms of the inconclusiveness of numbers, of statistics and number crunching in general you say some true things. I am however pretty certain that the subset of your thoughts and sub-claims that is clear and substantial is also constantly being evaluated (and **re-evaluated**) in the corresponding discussion rounds within the RKI, and within any agency consulting the German government. Most importantly, I am convinced that the case numbers presented in _this_ repository did **not** substantially contribute to decision-making (one might say: "fuck, we _know_ that these numbers are crap, for Christ's sake -- and yet we have to work with them -- tell us something new!").

You did not convince me that you can contribute something novel to the discussion.

I want to add an often underestimated aspect **about making decisions**: decision makers have a hard time making a decision in view of uncertainty. Yet, they _have to_ make decisions. Sometimes, it's better to make _a_ decision not knowing what's right and wrong, than making _no_ decision. Decision makers know this.

That said, from politics I **do** expect constant iteration in terms of decision-making, in view of new insights. Yes!

And I want to add an often underestimated aspect **about communicating decisions**: especially given _super complex topics_ it may be an unsolvable problem to communicate decisions in a way so that they satisfy all persons in a large audience. It's just such a difficult task. We should show some appreciation for that. Just try to imagine _your_ communication performance in press conferences after days/weeks of emergency-level work.

A good guideline for all of us: be skeptical, think positively, be optimistic, try to contribute in a productive way.

**Comment by nikola** [comment]
Honestly, I don't know what your point is, either. Is it maybe something like, "Let's not ask critical questions but stay the course and continue publishing numbers we all know are fraudulent because they are originating from points of authority?"

My own motivation was spelled out in the title of this ticket, "Confirmation of COVID-19 not possible", and in the first sentence, "there is actually no scientific test - as in: a provable and repeatable test - to confirm a case of COVID-19". And what followed was an invitation to follow my thought process how and why arrived at this conclusion.

I remember a time when it was a researcher's responsibility to prove that the numbers they're publishing are factual. Now you're basically claiming that I haven't disproven them to your satisfaction so you're going to continue publishing them and dismiss my critique altogether.

Well, ok. In my circles this is called lack of academic rigor.

**Comment by jgehrcke** [comment]
For the record, related: https://twitter.com/CT_Bergstrom/status/1243252341756669953

**Comment by nikola** [comment]
I don't know what the point of that Twitter thread is, either. Is it that the guy is surprised that life is difficult ("Every morning I wake up after 4-6 hours of fitful sleep"), or that other people have different opinions and he doesn't know how to cope with that?

Sorry, Jan, but at some point you've completely lost me. I honestly have no idea what you're trying to do here.

Anyway, feel free to ignore this ticket, no need to come back.

**Comment by gpaul** [comment]
> I repeat: right now there is no way to prove in any scientific sense that a person infected with SARS-CoV-2 and who also developed e.g. pneumonia is actually infected by COVID-19, or if they were "only" infected by SARS-CoV-2, fought off the infection by SARS-CoV-2 but developed the above symptoms due to an unrelated infection with other bacteria or viruses. Which would actually explain why the vast majority of people infected by SARS-CoV-2 survive without developing any of the above symptoms, and why the vast majority of confirmed or suspected COVID-19 cases had pre-existing conditions, namely conditions causing the exact same symptoms.

Are you suggesting that the overwhelming surge in severe pneumonia cases across the world is unrelated to the SARS-CoV-2 virus and that people with pre-existing conditions would experience these symptoms and outcomes regardless? That seems unreasonable. 

People with pre-existing conditions are exactly that: people who have been living with their conditions for some period of time, with varying degrees of medical support and placing a hitherto predictable burden on healthcare systems. 

At no point in the last decades were any hospitals in the developed western world overwhelmed by a sudden surge of people with pre-existing conditions contracting pneumonia in a period of a few weeks.

By qualitative reasoning it is far-fetched that a massive surge in terminal pneumonia has any cause outside viral infection. The difference between 10,000 cases and 30,000 cases, or X cases for that matter, is irrelevant: the only relevant fact is that healthcare systems are overwhelmed, and overwhelmed by people in respiratory distress and pneumonia.

Saying that people with pre-existing conditions are the ones hit hardest is tautological. Of course people with pre-existing conditions will comprise the highest mortality rates, that is true regardless of the symptoms a given disease exhibits. Indeed, it would be very surprising if for some reason Ebola or even some parasitic plague would effect people with pre-existing conditions to a lesser or equal degree than the general public.

We have no reason to think that the steady-state causes of pneumonia as seen in 2019 have increased in 2020 and certainly not exponentially. We also know that the cases _have_ increased exponentially, so there is some set of causes that appear to be increasing the spread of pneumonia exponentially. That set of causes leads to an exponential surge in pneumonia cases that overwhelms healthcare systems and we can safely call that surge COVID-19. We could call it anything else, too, if you prefer.

Just as we were happy to ignore baseline pneumonia in 2019, we can happily ignore "regular pneumonia" (as caused by conditions as prevailed in 2019) from the surge in 2020.

At this point, it is all but clear that the surge of pneumonia cases is due to infectious disease, and since it appears to correlate well with the spread of SARS-Cov-2, I struggle to see what remains to be said?

Concievably, some percentage of the clinical diagnoses being made for COVID-19 are done erroneously - the person could have regular pneumonia. But that amount of misdiagnoses shouldn't bother anyone in 2020, just like that mount of diagnoses for regular pneumonia didn't bother anyone in 2019. 

It seems very pedantic to harp on the name of the disease because there aren't direct tests for it, yet. Are you suggesting that we proceed as normal, for the sake of pedantry, until we can reliably test for a disease we can clearly see overwhelm our society?

The quantitative can be argued, and as @jgehrcke points out, we trust our data will improve empirically with time (unless we believe in a global conspiracy theory that harms every possible super power or illuminati group). But at no point does skepticism about the quantitative impact the present disaster of the qualitative. Not with this many qualitative examples to draw from.

**Comment by nikola** [comment]
@gpaul Your response is symptomatic, let me illustrate:

> Are you suggesting that the overwhelming surge in severe pneumonia cases across the world
> We also know that the cases have increased exponentially

So let's dissect:

* "overwhelming"
* "surge"
* "across the world"
* "increased exponentially"

Basically you're not building your argument around facts that you actually possess, but you just inject a dose of pessimistic speculation into the argument while providing no evidence at all. "Overwhelming"? "Surge"? "Across the world"? "Exponential increase"? What is this based on?

Let me rectify that assertion, and let's stay in Germany for a moment:

https://www.destatis.de/DE/Themen/Gesellschaft-Umwelt/Gesundheit/Todesursachen/_inhalt.html

In 2017, 932.272 people in Germany died. Out of those, 68.400 died from immediate, lung-related complications. To give you another reference point, 156.177 died from circulatory system diseases and complications.

As of this writing, the RKI claims that over a period of 4 weeks, 455 people died from COVID-19. Doing very basic math, and assuming a normal distribution, in a non-coronavirus year 68.400 / 12 = 5700 die every month from lung-related complications. Now, actually, the majority of those cases would appear in the month of March based on RKI's annual statistics, so the number 5700 would really be higher in March, but let's assume for a moment that it's not. So you have 455 deceased in March 2020, assuming that the cases were actually related to SARS-CoV-2 (and as I've shown above, the RKI is instructing stakeholders to count these cases against COVID-19 even in the absence of a positive test result for SARS-CoV-2), vs 5700 in March 2020 in non-SARS-CoV-2-related deaths.

So we're now finding out that your assertion about a "surge" is not only not true, but it's quite the opposite: in a non-coronavirus year in Germany, 12x as many people die in a single month from lung-related diseases than what we're currently seeing.

But let's take a look at New York, which is frequently cited as the new epicentre of the disease:

As of this writing, JHU claims that 1244 died from COVID-19 in the state of New York in March.

New York state has a population of approx. 19.453.561. New York city has a population of approx. 8.398.748. I'm pointing this out for statistical reference.

According to the CDC, here are the leading causes of death in New York state:

https://www.cdc.gov/nchs/pressroom/states/newyork/newyork.htm

Leaving aside heart diseases and diabetes for a moment, we see chronic lower respiratory diseases (7.258) plus flu/pneumonia (4.517), with a total of 11.775 deaths in New York state caused by the two. That's approx 980 deaths per month, assuming normal distribution.

So how would 1.244 claimed COVID-19 deaths in New York state in March 2020 indicate a "surge" in any sense against 980 deaths due to lung-related complications in New York state in regular months?

And I'm actually being very favorable to your side of the argument because above I'm not even counting heart diseases, cancer and diabetes, which, according to the WHO and CDC, are also pre-existing conditions that contribute to a high mortality rate of SARS-CoV-2 infections. If I added those two causes, we'd be looking at 94.999 deaths per year caused by the above, or 7916 per month.

So that's 7916 for any given month, vs 1244 in March 2020. And, again, neither the CDC nor the JHU are actually claiming that COVID-19 was the single cause of death in the 1244, but that they died _and_ that they were infected by SARS-CoV-2. There's a difference between "dying with COVID-19" and "dying due to COVID-19", a distinction apparently lost among journalists and politicians.

Of course often I read an argument that goes something like, "But you're comparing deaths that would've occured due to normal illnesses to deaths of healthy people", an argument that is propped up by articles that downplay pre-existing conditions but claim that someone who allegedly had no pre-existing conditions died from COVID-19. But here the same counter-argument applies: this assertion is based on no evidence whatsoever because surely none of the people making this claim have or had access to the very private medical records of the deceased to determine that they really had no pre-existing conditions.

Another argument I'm seeing often is that we'd be only at the beginning of a surge and that, surely, next week or in two weeks or in a month or two months the tsunami would hit us. (Which is actually a term frequently used by politicians and journalists to whip up hysteria.) But isn't it interesting that this tsunami was already predicted 2 weeks ago to be happening this week, and that it's precisely this week where suddenly many scientists, including Dr Christian Drosten who is so revered by many in Europe, decide they want to take some time off from public appearances?

And then at some point the discussion becomes circular: the counter-argument is then made that it was only the harsh measures of social distancing that prevented the tsunami, and when you point out why the measures are still in place the argument is reversed into: because of the tsunami.

In other words, most arguments I'm seeing are not based on scientific proof or even just evidence, but on fear and ignorance of historical numbers.

**Comment by gpaul** [comment]
>  "overwhelming", "surge", "across the world", "increased exponentially"

I use those terms because I assumed we weren't arguing that there is a surge and that the number of cases of people needing to be hospitalized for pneumonia is increasing exponentially.

It seems that we are arguing that, and that the crux really is whether or not hospitals are being overwhelmed or show trajectories that suggest they will be without the extreme measures being taken.

Since that is the case, and you provide statistics suggesting that hospitals are not being overwhelmed, or at least that since mortality rates seem typical, that hospitals are successfully dealing with any supposed surge, the burden of proof is on me to respond with statistics to the contrary.

This is the crux of conspiracy: I'd need to find or compile up-to-date statistics on the occupancy of hospitals in various countries to compare how different strategies (lockdown, social distancing, or prayer) correlate to how overburdened their healthcare systems are. Or, I can trust that those in positions of power stand to gain nothing by fabricating the crisis and shutting down global economy.

Personally, given my limited free time, I find that I trust the reports of rapidly overburdened healthcare systems in Wuhan and Italy good enough for me to live with the possibility that we are being lied to about a surge in pneumonia cases unprecedented in recent decades.

I would love to see those statistics though: how many pneumonia cases reported per month over the last 3 months and the same period in previous years, for Italy, Wuhan, Spain, etc.

**Comment by nikola** [comment]
You're missing the point. It's not whether any given numbers are high or low in isolation.

The point is whether any of the numbers making the rounds are exceptional. They are not.

And nobody said that the crisis is fabricated. What's implausible is the claim that these numbers are so exceptional that we must take drastic measures not seen before in modern Western societies.

In 2017, the RKI - the same RKI producing the official COVID-19 case numbers in Germany - published that more than 25.100 people in Germany died from the flu in that season alone, ie. of complications and secondary infections caused by the flu. It was a severe flu season.

In comparison 2018 was a mild flu season with "only" a few hundred fatalities. However, more than 40.000 people in Germany were hospitalized because of the flu, and more than 5 million doctors visits were done that year. During a mild flu season. In 2017 it was double that number. When the COVID-19 pandemic is under control, how many people will have been hospitalized in Germany because of COVID-19? Isn't it curious that hardly any institution is publishing how many COVID-19 patients there are at any given point in time? You'd think that when they know how many people were tested positive or how many died then they'd also know how many are currently under treatment, no? And isn't it curious that you hardly see any numbers published about the severity of each COVID-19 case? All that is ever published, by journalists, are anecdotes how hospitals are exceeding capacity. So what is the total capacity available, and by how much is it exceeded?

But since you mentioned Italy, in 2017 in Italy more than 24.000 died from the flu:

https://www.sciencedirect.com/science/article/pii/S1201971219303285

Quotes:

* In the winter seasons from 2013/14 to 2016/17, an estimated average of 5,290,000 ILI cases occurred in Italy, corresponding to an incidence of 9%.
* More than 68,000 deaths attributable to flu epidemics were estimated in the study period.
* Italy showed a higher influenza attributable excess mortality compared to other European countries. especially in the elderly.
* Over 68,000 deaths were attributable to influenza epidemics in the study period. The observed excess of deaths is not completely unexpected, given the high number of fragile very old subjects living in Italy. 
* UK estimates, in terms of absolute numbers, were higher compared to Italian data, in all ages and in particular in the elderly (26,542 vs 19,475 respectively).
* In terms of amplitude of the at risk population, in Italy there are 6.7 million of people aged 75+ (more than 10% of the population) that constitute a large group of fragile subjects, among which the annual death rate is naturally high, around 4%.

I'm not seeing a conspiracy here. I don't think that any nurse or doctor in Italy is conspiring to fake anything, the suffering is real. But so was the suffering in 2013, 2014, 2015, 2016, and the same nurses and doctors in Italy were seeing hospitals overwhelmed.

Ironically, the exact same people who are now saying that this is not just another flu are also the people who frequently cite the Spanish Flu - you know, the one where everybody is now claiming killed millions of people when in fact it was the problem that antibiotics haven't been invented back then and most fatalities were from secondary, bacterial infections accelerated by weakness after a long world-war - as evidence how a virus can kill millions of people and that we should all prepare for extinction.

So now we're back to why I created this ticket: the data points published in this repository do not represent what they claim they'd represent. They are not tracking the spread of the SARS-CoV-2 infection, they are not tracking significant cases of COVID-19, and the data points are not comparable - you know, the quality required to create meaningful statistics - because they do not show when tests were done.

The remedy? Public pressure on the authorities to release that information.

> Or, I can trust that those in positions of power

and that is precisely why I disagree with you.
